terraform/production/main.tf

```diff
@@ -77,7 +77,7 @@ module "auth_service" {
     from_name        = "Delta System"
     from_address     = "delta@datacollection.levellingup.gov.uk"
     reply_to_name    = "DLUHC Digital Services"
```

Comment:
These should also be updated.

Reply:
From address can't be updated yet, but yeah may as well update the name.